app/Models/Post.php

```diff
@@ -275,7 +275,8 @@ public function toBlinkPayload()
         $campaign_cause = optional($this->signup->campaign)->getAttributes()['cause'];
 
         // Fetch Campaign Website information via GraphQL.
-        $campaignWebsite = app(GraphQL::class)->getCampaignWebsiteByCampaignId($this->campaign_id);
+        // $campaignWebsite = app(GraphQL::class)->getCampaignWebsiteByCampaignId($this->campaign_id);
```

Comment:
Should this line be removed?

Reply:
Gah, yes!! 😖 I have to remove that to fix an issue communicating with the local GraphQL server when developing (need a better fix!) and forgot to uncomment here.